tests/parser/functions/test_sha3.py

```diff
@@ -11,9 +11,9 @@ def bar() -> bytes32:
 
     c = get_contract_with_gas_estimation(hash_code)
     for inp in (b"", b"cow", b"s" * 31, b"\xff" * 32, b"\n" * 33, b"g" * 64, b"h" * 65):
-        assert c.foo(inp) == utils.sha3(inp)
+        assert '0x' + c.foo(inp).hex() == sha3(inp).hex()
 
-    assert c.bar() == utils.sha3("inp")
+    assert '0x' + c.bar().hex() == sha3(b"inp").hex()
```

Comment:
Reduces readability on the assert statement but you could drop the `hex` overhead and do the direct assertion.